### PR TITLE
Manjaro WIP

### PR DESCRIPTION
**Pull request by Invertisment**
This is my try to port it. I didn't try to make it pretty but this is what I have. And it doesn't work.

`wheel` is not needed but I wanted to compile `PyGObject` so I added it (somebody suggested it).

It fails when it tries to build `PyGObject`.

This OS package:
`gobject-introspection-1.70.0-2-x86_64` fails to install because every mirror responds with 404.
(`sudo pacman -S gobject-introspection`)

And if I don't install it then I get this error (from `pip`):
```
  running build_ext
  Package gobject-introspection-1.0 was not found in the pkg-config search path.
  Perhaps you should add the directory containing `gobject-introspection-1.0.pc'
  to the PKG_CONFIG_PATH environment variable
  Package 'gobject-introspection-1.0', required by 'virtual:world', not found
  Command '('pkg-config', '--print-errors', '--exists', 'gobject-introspection-1.0 >= 1.56.0')' returned non-zero exit status 1.
  ----------------------------------------
  ERROR: Failed building wheel for PyGObject
Failed to build PyGObject
ERROR: Could not build wheels for PyGObject, which is required to install pyproject.toml-based projects
```

More details:
https://github.com/salihmarangoz/system_tray_extensions/issues/38#issuecomment-966323057

### CODE REVIEW

File: requirements_pacman.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Source: https://github.com/gfduszynski/cm-rgb/issues/45#issuecomment-920936950
+sudo pacman -S \
+  cairo \
+  gobject-introspection \
```
Comment (Invertisment) [inline requirements_pacman.sh:5]: This fails with 404

File: requirements_pip.txt [hunk 2]
```diff
@@ -11,4 +11,5 @@ gitpython
 psutil
 matplotlib
 soundcard
-scipy
\ No newline at end of file
+scipy
+wheel
```
Comment (Invertisment) [inline requirements_pip.txt:15]: Added because I tried to build `PyGObject`

File: manjaro-install.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+#INSTALL_DIR="$SCRIPT_DIR"
+INSTALL_DIR=$(pwd)
+#INSTALL_DIR="$HOME/.system_tray_extensions"
+
+####################################################
+echo "2. Create new virtual environment"
+sudo pacman -S python-virtualenv
```
Comment (Invertisment) [inline manjaro-install.sh:13]: This is not `python3.8-virtualenv` but I think it should work

**Comment by salihmarangoz** [comment]
I got it working with a live Manjaro USB! Will be updating the installation. I will notify you in the issue then you can test it.

**Comment by salihmarangoz** [comment]
@Invertisment I have updated the installation. Read the Installation section in the README.md again. In general, `base-devel` meta package solved the issue on Manjaro. I think it is related to development tools, etc.

File: requirements_pacman.sh (hunk 1 above)
Comment (salihmarangoz) [reply requirements_pacman.sh:5]: I have tested with Manjaro 21.1.6 live boot environment and it as working. Is it still a problem for you?

Comment (Invertisment) [reply requirements_pacman.sh:5]: I have Manjaro `21.2.0` and for some reason the package didn't install.
I'll try again.

Comment (Invertisment) [reply requirements_pacman.sh:5]: I found my bug.
I had switched Manjaro branches from stable to unstable. Then back. And I didn't update the repository links.
Turns out that my scripts were working :tada: :+1:

Comment (Invertisment) [reply requirements_pacman.sh:5]: Confirmed. Works.

**Pull request merged**